nengo/node.py

```diff
@@ -7,10 +8,21 @@
 from nengo.exceptions import ValidationError
 from nengo.params import Default, IntParam, Parameter
 from nengo.processes import Process
-from nengo.utils.compat import is_array_like
+from nengo.utils.compat import is_array_like, getfullargspec
 from nengo.utils.stdlib import checked_call
 
 
+class OutputFnArgsValidationError(ValidationError):
```

Comment:
I'm all for OOP, but can we just have a function that does this? From what I can tell, it's just a helper for making the `msg`.

Reply:
Otherwise it'll show up as `OutputFnArgsValidationError` in error traces, which is a little weird to me.